compiler/x/codegen/FPTreeEvaluator.cpp

```diff
@@ -453,7 +453,7 @@ TR::Register *OMR::X86::TreeEvaluator::fpBinaryArithmeticEvaluator(TR::Node
                                                               bool              isFloat,
```

Comment:
Is there any point to this function?  Why not have its callers simply call `floatingPointBinaryArithmeticEvaluator()`.  The `isFloat` parm is eaten anyway.